lib/sync-util.js

```diff
@@ -36,19 +52,32 @@ var sortObject = function (object) {
   return result;
 }
 
+var asyncSortedStringify = function (obj, cb) {
+  console.log('asyncSortedStringify');
+  console.log(typeof obj);
+  console.log(obj);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)